network-api/networkapi/nav/models.py

```diff
@@ -170,7 +170,7 @@ def localized_featured_blog_topics(self):
         blog_index_page = self.blog_index_page
         for relationship in featured_topics_relationships:
             if relationship.topic:
-                relationship.topic.url = blog_index_page.reverse_subpage(
+                relationship.topic.url = blog_index_page.url + blog_index_page.reverse_subpage(
```

Comment:
I always forget that `reverse_subpage` doesn't give a full url, only the last bit :man_facepalming: you still have to add the base from the routable page to it